src/allotropy/allotrope/schema_mappers/adm/cell_counting/rec/_2024/_09/cell_counting.py

```diff
@@ -0,0 +1,337 @@
+                    for measurement_group in data.measurement_groups
+                ],
+            ),
+        )
+
+    def _get_technique_document(
+        self, measurement_group: MeasurementGroup, metadata: Metadata
+    ) -> CellCountingDocumentItem:
+        return CellCountingDocumentItem(
+            analyst=measurement_group.analyst,
+            measurement_aggregate_document=MeasurementAggregateDocument(
+                measurement_document=[
+                    self._get_measurement_document(measurement, metadata)
+                    for measurement in measurement_group.measurements
+                ]
+            ),
+        )
+
+    def _get_measurement_document(
+        self, measurement: Measurement, metadata: Metadata
+    ) -> MeasurementDocument:
+        return MeasurementDocument(
+            measurement_time=self.get_date_time(measurement.timestamp),
+            measurement_identifier=measurement.measurement_identifier,
+            sample_document=self._get_sample_document(measurement),
+            device_control_aggregate_document=DeviceControlAggregateDocument(
+                device_control_document=[
+                    DeviceControlDocumentItem(
+                        device_type=metadata.device_type,
+                        detection_type=metadata.detection_type,
+                        sample_volume_setting=quantity_or_none(
+                            TQuantityValueMicroliter, measurement.sample_volume_setting
+                        ),
+                    )
+                ]
+            ),
+            processed_data_aggregate_document=self._get_processed_data_aggregate_document(
+                measurement
+            ),
+            error_aggregate_document=self._get_error_aggregate_document(
+                measurement.errors
+            ),
+        )
+
+    def _get_sample_document(self, measurement: Measurement) -> SampleDocument:
+        # TODO(ASM gaps): we believe these values should be introduced to ASM.
+        custom_document = {
+            "group identifier": measurement.group_identifier,
+            "sample draw time": (
+                self.get_date_time(measurement.sample_draw_time)
+                if measurement.sample_draw_time
+                else None
+            ),
+        }
+        return add_custom_information_document(
+            SampleDocument(
+                sample_identifier=measurement.sample_identifier,
+                batch_identifier=measurement.batch_identifier,
+                written_name=measurement.written_name,
+            ),
+            custom_document,
+        )
+
+    def _get_processed_data_aggregate_document(
+        self, measurement: Measurement
+    ) -> ProcessedDataAggregateDocument:
+        # TODO: average values and stddev should be calculated data
+        # TODO(ASM gaps): we believe "total object count" and "aggregate rate" should be introduced to ASM.
+        custom_document = {
```

Comment:
Seems like most of this (averages, std dev) should actually be moved to calculated data document (these were not available in previous schema)

Can you work with @james-leinas to determine the right representation for these?

Reply:
Will do!

Reply:
@nathan-stender The fields that you mentioned are actually only included in the `Roche Cedex HiRes` adapter. I suggest we merge this PR and then I'll make the changes along with the migration of that adapter to this new mapper.